nautobot-app/hooks/post_gen_project.py

```diff
@@ -15,7 +15,7 @@
 * poetry install
 * poetry shell
 * invoke makemigrations
-* ruff . # this will ensure all python files are formatted correctly, may require `sudo chown -R <my local username> ./` as migrations may be owned by root
+* ruff check . # this will ensure all python files are formatted correctly, may require `sudo chown -R <my local username> ./` as migrations may be owned by root
```

Comment:
Given the comment and the fact that invoke was used for makemigrations, I thought that there might have been a reason ruff was done locally instead of through invoke. They should have ruff through the `poetry install` command above.

Reply:
I had completely missed the `poetry shell` above.